### PR TITLE
changed tittle and added css file

### PR DESCRIPTION
**Pull request by Scecin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html lang="es">
 <head>
-    <title>Masterclass de Git</title>
+    <title>Masterclass de Gat</title>
```
Comment (nalcalag) [inline index.html:4]: NIT: I would change it back to Git

**Review by nalcalag — CHANGES_REQUESTED** [review]
Please remove the css file if you are not using it

File: index.html (hunk 1 above)
Comment (Scecin) [reply index.html:4]: It needs to be like that

**Review by nalcalag — APPROVED** [review]
Good job! 🎉

**Pull request merged**